src/SFA.DAS.Admin.Roatp.Web/Controllers/AddProvider/SelectProviderController.cs

```diff
@@ -56,13 +55,23 @@ public async Task<IActionResult> Index(SelectProviderSubmitModel submitModel, Ca
             return View(viewModel);
         }
 
+        if (organisationApiResponse.StatusCode != System.Net.HttpStatusCode.OK && organisationApiResponse.StatusCode != System.Net.HttpStatusCode.NotFound)
```

Comment:
`organisationApiResponse.StatusCode != System.Net.HttpStatusCode.OK` is redundant check